packages/vm/src/runBlock.ts

```diff
@@ -255,20 +257,19 @@ export async function runBlock(vm: VM, opts: RunBlockOpts): Promise<RunBlockResu
     block = createBlock(blockData, { common: vm.common })
```

Comment:
`requests` is no longer a field in `BlockData`, so what is the relevance of passing `requests` into `createBlock` here?

Reply:
yes, it should be removed, must been missed